README.docker.md

```diff
@@ -0,0 +1,32 @@
+# Staytus
+
+Staytus is a complete solution for publishing the latest information about
+any issues with your web applications, networks or services. Along with
+absolutely beautiful public & admin interfaces, Staytus is a powerful tool for
+any organization with customers that rely on them to be online 24/7.
+
+* [Check out the live demo](http://demo.staytus.co)
+* [Read the roadmap](https://github.com/adamcooke/staytus/blob/master/ROADMAP.md)
+* [Report a bug](https://github.com/adamcooke/staytus/issues/new?labels=bug)
+* [Ask a question](https://github.com/adamcooke/staytus/issues/new?labels=question)
+
+![Screenshot](https://s.adamcooke.io/15/iOzvtk.png)
+
+## Setup with Docker
+
+1. [Install Docker locally](https://docs.docker.com/installation/)
+2. [Install docker-compose locally](https://docs.docker.com/installation/)
+3. Now run `docker-compose -d up`, to start Staytus with the database container (You have to be in the directory of your Staytus `docker-compose.yml` for this).
+4. Go to [http://localhost:80](http://localhost:80) and follow the instructions to configure Staytus
```

Comment:
I think this needs you to have a checkout of the codebase, or at least a local copy of the docker-compose.yml, right? One of the nice things with the current setup is that you don't have to do that; you just ask docker to run staytus, and then you're done. Looks like there isn't yet a neat solution to this in Docker (https://github.com/docker/compose/issues/1818), but an initial one's en route imminently: https://github.com/docker/compose/pull/3098.

For now, I think just add a step to clone the codebase locally, or a direct link to download the docker-compose here, so people know what's required.

As soon as that PR is merged and released though we should recommend just:

* Install docker
* Install docker-compose
* Run `docker-compose -f https://github.com/adamcooke/staytus/raw/master/docker-compose.yml up`